xilem_core/src/message.rs

```diff
@@ -90,15 +88,66 @@ impl DynMessage {
     }
 }
 
+// We could consider:
+// ```
+// enum DynMessage {
+//     Special(Box<dyn AnyMessage>),
+//     Send(SendMessage)
+// }
+// ```
+// to let "maybe-threaded" message handling. That would be especially useful for
+// handling stale messages (i.e. reporting them back to the task which failed).
+// Probably not worth it, but would be (reasonably) non-breaking, at least.
+// Ooh, we could pass a `fn(DynMessage)->Option<SendMessage>`, which would assume/validate that
```

Comment:
"Ooh" intentionally? Or was "Or" meant?

I've no meaningful comment though yet on the comment itself...